doc/subsystems/networking/usbnet_setup.rst

```diff
@@ -0,0 +1,79 @@
+.. _usb_device_networking_setup:
+
+USB Device Networking
+#####################
+
+This page describes how to set up networking between a Linux host
+and a Zephyr application running on USB supported devices. A sample
+application from Zephyr source distribution is run on one of
```

Comment:
Change to:

    The :ref:`echo-server-sample` application from the Zephyr source distribution
    is run on supported board.  The board
    is connected to a Linux host using a USB cable providing an Ethernet
    interface to the host.

Reply:
I will change to this but note that there are many sample applications and echo-server only one of them.